spec/models/converters/date_converter_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe DateConverter do
+  subject { described_class }
+
+  it 'converts strings to dates' do
+    expect(subject.convert('4/17/2017')).to eq(Date.parse('April 17, 2017'))
+  end
+
+  it 'converts blank value to nil' do
+    expect(subject.convert('')).to be_nil
+  end
+
+  # TODO: we should find a way to pass warnings off for data that doesn't parse
+  it 'converts bad date to nil' do
```

Comment:
is this TODO comment something that needs to be addressed prior to merge?

Reply:
I think we'll need to rethink the approach to the converters in order to surface these errors, so something that needs to be done later.